AniHubApp/AniHubApp/Models/AnimeGenre.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace AniHubApp.Models
+{
+    public class AnimeGenre : INotifyPropertyChanged
```

Comment:
It's not necessary to implement `INotifyPropertyChanged`, Fody takes care of the heavy work